nzediegwu1/EventManager
# #157733863 Implement pagination

#### What does this PR do?
Implement pagination

#### Description of Task to be completed?
1. Implement server side pagination for event list
2. Implement server side pagination for center list
3. Implement server side pagination for user list
4. Implement functionality to handle pagination in eventList component
5. Implement functionality to handle pagination in centerList of addEvent component
6. Define redux action and constant getEventCenters and GET_EVENT_CENTERS respectively
7. Define redux action and constant setDataCount and SET_DATA_COUNT respectively
8. Set pagination LIMIT constant globally
9. Implement functionality to retrieve center selected in addEvent component
10. Handle SET_DATA_COUNT and GET_EVENT_CENTERS in page and center reducers
11. Add scss styling for pagination feature
12. Modify API json response for easy integration with frontend pagination feature
13. Install react-js-pagination, babel-presets-es2015 and stage-0 dependencies
14. Implement functionality to handle pagination in userList component
15. Implement functionality to handle pagination in centerList component

#### How should this be manually tested?
1. On terminal, run command: `npm run start:dev` and `npm run client`
2. On Postman, enter in users, centers and events routes, variations of the following query: pageNumber=2&limit=3
3. Verify that it works as expected

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories? (If applicable)
[#157733863](https://www.pivotaltracker.com/story/show/157733863)

#### Screenshots (If applicable)
